components/build/build-service/.tekton/kustomization.yaml

```diff
@@ -0,0 +1,15 @@
+resources:
+- serviceaccount.yaml
+- pipelines-as-code.yaml
+- pvc.yaml
+
+namespace: build-service-system
```

Comment:
I think the namespace should be `build-service`.
The default `build-service-system` is generated by operator-sdk, but in our components we don't usually use the `system` suffix: `application-service` namespace is an example.